spec/page_objects/application_page.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```

Comment:
Just an Idea but it has been really helpful having a template site prism and extend these classes from it. Examples are on the gitlab national-library and kakapo applications.